### PR TITLE
Master <> Develop Sync (post-v10.8.0)

### PR DESCRIPTION
**Pull request by danjm**
Syncs master with develop (post v10.8.0)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,13 +6,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- [#12802](https://github.com/MetaMask/metamask-extension/pull/12802): Fix bug causing occasional swaps failures for token pairs with highly precise exchange rates
+- [#12679](https://github.com/MetaMask/metamask-extension/pull/12679): Ensure eth_sign callback fires even when data submitted is invalid
+
 ## [10.7.1]
 ### Fixed
 - [#13005](https://github.com/MetaMask/metamask-extension/pull/13005): Fix connection to RPC urls with query strings.
 
 ## [10.7.0]
 ### Changed
-- [#12643](https://github.com/MetaMask/metamask-extension/pull/12643): Reject popup confirmations on close
```
Comment (Gudahtt) [inline CHANGELOG.md:15]: Any idea why this was removed?

**Review by Gudahtt — APPROVED** [review]
LGTM!

File: CHANGELOG.md (hunk 1 above)
Comment (danjm) [reply CHANGELOG.md:15]: no... I'll look into this...

Comment (danjm) [reply CHANGELOG.md:15]: It looks like this was a mistake when I made the v10.8.0 changelog commit https://github.com/MetaMask/metamask-extension/commit/0563c471a66f07b882f1a69c1c340bd9aeb59ddd#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

Comment (danjm) [reply CHANGELOG.md:15]: I guess we should merge this PR and then create a separate PR to fix that mistake?

Comment (Gudahtt) [reply CHANGELOG.md:15]: That would work, sure!

**Review by jpuri — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**